### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by h1deeeee**
- [DEMO LINK](https://h1deeeee.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://h1deeeee.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,70 @@
...
+
+      <h2 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card_stars-reviews">
```
Comment (vlad-demchuk) [inline src/index.html:33]: Use BEM naming convention for 'card_stars-reviews', change it to 'card__stars-reviews'.

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,70 @@
...
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
```
Comment (vlad-demchuk) [inline src/index.html:38]:
```suggestion
          <div class="stars__star"></div>
```
no need to add blanks between one-line elements

File: src/styles/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+.card {
+  display: flex;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  align-items: center;
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.css:14]: do not forget about css properties order
http://codeguide.academy/html-css.html#css-order

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,66 @@
...
+<html lang="en" class="page-card">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
```
Comment (DarkMistyRoom) [inline src/index.html:11]:
```suggestion
      content="
        width=device-width,
        user-scalable=no,
        initial-scale=1.0,
        maximum-scale=1.0,
        minimum-scale=1.0
      "
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,66 @@
...
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+
+      <h2 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <span class="card__product-code">
+        Product code: 195434
+      </span>
```
Comment (DarkMistyRoom) [inline src/index.html:31]:
```suggestion
      <p class="card__product-code">
        Product code: 195434
      </p>
```

Comment (DarkMistyRoom) [reply src/index.html:31]: better use semantic tags

Comment (DarkMistyRoom) [reply src/index.html:31]: change it everywhere

File: src/styles/blocks/card.css [hunk 6]
```diff
@@ -0,0 +1,96 @@
+.page-card {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
```
Comment (DarkMistyRoom) [inline src/styles/blocks/card.css:4]: no need to create BEM block for font settings. You can use just `html `tag

File: src/styles/blocks/card.css [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+  color: var(--main-color-card);
+}
+
+.card__price {
+  display: flex;
+  width: 100%;
+  margin-bottom: 16px;
+  justify-content: space-between;
+
+}
```
Comment (DarkMistyRoom) [inline src/styles/blocks/card.css:62]:
```suggestion
  justify-content: space-between;
}
```

File: src/styles/reset.css [hunk 8]
```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```
Comment (DarkMistyRoom) [inline src/styles/reset.css:4]: Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**